Content.Shared/Language/LanguagePrototype.cs

```diff
@@ -10,15 +9,13 @@ public sealed class LanguagePrototype : IPrototype
     public string ID { get; private set;  } = default!;
 
     /// <summary>
-    ///     If true, obfuscated phrases of creatures speaking this language will have their syllables replaced with "replacement" syllables.
-    ///     Otherwise entire sentences will be replaced.
+    ///     Obfuscation method used by this language. By default, uses <see cref="ObfuscationMethod.Default"/>
     /// </summary>
-    [DataField(required: true)]
-    public bool ObfuscateSyllables;
+    [DataField("obfuscation")]
+    public ObfuscationMethod Obfuscation = ObfuscationMethod.Default;
 
     /// <summary>
-    ///     Lists all syllables that are used to obfuscate a message a listener cannot understand if obfuscateSyllables is true.
-    ///     Otherwise uses all possible phrases the creature can make when trying to say anything.
+    ///     A list of replacement phrases used in <see cref="ObfuscationMethod"/>
     /// </summary>
     [DataField(required: true)]
     public List<string> Replacement = [];
```

Comment:
Now that language obfuscators are custom classes, this should be moved to the two that actually use it, since not all languages will.